src/Discord.Net.Core/Entities/Guilds/IGuildIntegration.cs

```diff
@@ -1,17 +1,39 @@
-﻿using System;
+using System;
 
 namespace Discord
 {
     public interface IGuildIntegration
     {
+        /// <summary> Gets the integration ID. </summary>
+        /// <returns> Gets the integration ID. </returns>
```

Comment:
Pardon, but you might have misunderstood what I meant by the last request. `summary` and `returns` should not state the same thing. `summary` summarizes the member's purpose and what it does. `returns` explains what object is being returned and what for. 

For example:

```cs

        /// <summary> Gets the darker grey color value. </summary>
        /// <returns> A color struct with the hex value of <see href="http://www.color-hex.com/color/607D8B">607D8B</see>.</returns>
        public static readonly Color DarkerGrey = new Color(0x546E7A);
```

Reply:
Thanks for the clarification and the example.